File: Change Bits - GFG/change-bits.cpp
// { Driver Code Starts
//Initial Template for C++

#include <bits/stdc++.h>
using namespace std;

 // } Driver Code Ends
//User function Template for C++

class Solution
{
public:
    vector<int> changeBits(int N)
    {
        if (N == 0)
            return {1, 1}; 
        int power = log2(N) + 1;
        int newNum = (1ll << power) - 1;
        return {(newNum - N), newNum};
    }
};

// { Driver Code Starts.
int main() {
    int t;
    cin >> t;
    while (t--) {
        int N;
        cin>>N;
        Solution ob;
        auto ans = ob.changeBits(N);
        cout<<ans[0]<<" "<<ans[1]<<endl;
    }
    return 0;
}  // } Driver Code Ends

File: Change Bits - GFG/change-bits.py
#User function Template for python3
import math

class Solution:
    def changeBits(self, N):
        K=int(math.log2(N))
        K=2**(K+1)-1
        return [K-N,K]

#{ 
#  Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__': 
    t = int (input ())
    for _ in range (t):
        N=int(input())
        
        ob = Solution()
        ans = ob.changeBits(N)
        
        print(ans[0],ans[1])
# } Driver Code Ends